src/ThreeEditor/js/Viewport.js

```diff
@@ -482,7 +484,7 @@ export function Viewport(
 
         transformControls.detach();
 
-        if (object !== null && object !== scene && object !== camera) {
+        if (object !== null && object !== scene && object !== camera && object !== zones && !object?.unionOperations) {
```

Comment:
```suggestion
        // Check if object can be transform. 
        // For our usage it would be only geometries included on the scene. 
        // Amount of geometries can differ form project to project thus we check only if it isn't mesh.
        // unionOperations is property unique to zones that shoudn't be transformed with controler.
        if (object !== null && object !== scene && object !== camera && object !== zones && !object?.unionOperations) {
```